lib/models/pr.js

```diff
@@ -153,70 +155,74 @@ class PR {
     get payload() {
         return this._payload;
     }
-        
+
     set payload(payload) {
         return this._payload = payload;
     }
 
     get installation() {
         return this._installation;
     }
-    
+
     get number() {
         return this._number;
     }
-    
+
     get owner() {
         return this._owner;
     }
-    
+
     get repo() {
         return this._repo;
     }
-    
+
     get id() {
         return this._id;
     }
-    
+
     get body() {
         return this._body = this._body || (this.payload.body || "").replace(/\r\n/g, "\n").trim();
     }
-    
+
     get processor() {
         return this.config.type.toLowerCase();
     }
-    
+
     get aws_s3_bucket() {
         if (process.env.ALLOW_MULTIPLE_AWS_BUCKETS == "no") {
             return process.env.AWS_BUCKET_NAME;
         }
         return this.owner == "whatwg" ? process.env.WHATWG_AWS_BUCKET_NAME : process.env.AWS_BUCKET_NAME;
     }
-    
+
     get isMultipage() {
         return !!this.config.multipage;
     }
 
     requiresPreview() {
         return !(/<!--\s*no preview\s*-->/.test(this.body));
     }
-    
+
+    async requestIncludes() {
+        this.includes = await scanIncludes(Head.fromPR(this).github_repo_url, this.config.src_file, this.config.type);
```

Comment:
It feels weird to have includes also include the src file itself.

I would suggest either not including it in the returned set or renaming it.

Reply:
I could assign this directly to `this.relevantSrcFiles` and remove that property's getter?

Reply:
Sure. Don't we need includes separate from the main src file, though? E.g. to send them to Bikeshed/Respec?

Also what if we want to reuse this infra to handle images too, won't these need to be treated separately (e.g. so we can upload them to the S3 instance?

Reply:
It sounds like you'd rather have this omit the main file, so I've done that.

A scan for non-include subresources will need to return them as a separate set from the included files.

Reply:
Yeah, I think I prefer that. Good call.